README.rst

```diff
@@ -54,6 +54,160 @@ New content will be indexed in ElasticSearch.
 
 To index existing content, a full ``Clear and Rebuild`` is needed (via ZMI/``portal_catalog``/Tab ``Advanced``).
 
+ES Python dependencies
+----------------------
+
+Current version of this package requires elasticsearch_dsl. It is
+necessary to add the 'elasticsearch-dsl' egg to the buildout eggs.
+Alternatively, it can be added to the eggs in the celery part.
+Run the buildout again to get that dependency on existing installations.
+
+ES configuration on zope.conf
+-----------------------------
+
+The elasticsearch directive now supports the following keys:
+
+- max_blobsize. Max length of files to index, in bytes. If a file is
+  larger than this size, it will not be indexed, and this will be logged.
+  Default is zero, which means index everything.
+
+- request_timeout. The default connection timeout is 10 seconds. Using this
+  key it can be set to any number of seconds.
+
+- use_celery. If true, indexing will be done in async celery tasks. This
+  requires that celery is correctly configured.
+
+- indexed_chars. Maximum number of characters to extract from attachments.
+  Default is 100,000. Use -1 for infinite.
+
+Example:
+
+zope-conf-additional =
+    %import collective.es.index
+    <elasticsearch>
+    query 127.0.0.1:92000
+    ingest 127.0.0.1:92000
+    request_timeout 20
+    max_blobsize 10000000 # 10 MB
+    indexed_chars 200000
+    use_celery true
+    </elasticsearch>
+
+It is necessary to add this configuration to the buildout and rerun it
+whenever a change is made to these parameters.
+
+Celery configuration
+--------------------
+
+NOTE: The configuration now uses collective.celery, so it has changed.
+
+The collective.celery package requires adding the celery and
+collective.celery eggs to the mian buildout section eggs. Example:
+
+eggs =
+    celery
+    Plone
+    elasticsearch
+    elasticsearch-dsl
+    collective.es.index
+    collective.celery
+
```

Comment:
Same here, I would like to see this as `install_requires` and  `extra_requires` in `setup.cfg`.